.travis.yml

```diff
@@ -11,50 +11,56 @@ env:
     - PHPUNIT_FLAGS="-v"
     - PHPUNIT_ENABLED="true"
     - COVERALLS_ENABLED="false"
+    - IMAGICK_ENABLED="true"
+    - ENQUEUE_ENABLED="true"
     - STABILITY=stable
 
 matrix:
   fast_finish: true
   include:
     - php: '7.1'
     - php: '7.2'
+    - php: '7.3'
+    - php: '7.4'
 
     # Enable code coverage with the latest supported PHP version
-    - php: '7.3'
+    - php: '8.0'
       env:
         - COVERALLS_ENABLED="true"
+        - IMAGICK_ENABLED="false"
+        - ENQUEUE_ENABLED="false"
         - XDEBUG_MODE=coverage
         - PHPUNIT_FLAGS="-v --coverage-text --coverage-clover var/build/clover.xml"
 
     # Minimum supported dependencies with the latest and oldest supported PHP versions
     - php: '7.1'
       env:
         - COMPOSER_FLAGS="--prefer-lowest"
-    - php: '7.3'
+    - php: '8.0'
       env:
-        - SYMFONY_PHPUNIT_VERSION="7.5"
+        - SYMFONY_PHPUNIT_VERSION="8.5"
```

Comment:
You can remove this line and let PHPUnit Bridge choose the right version (see https://symfony.com/doc/current/components/phpunit_bridge.html#modified-phpunit-script)

Reply:
Isn't the ratio to try testing with older versions?

Reply:
i suggested fixing to phpunit 8 for now, because likely there are some assertions that will stop working with phpunit 9. that way we can keep upgrading to phpunit 9 separate, to not add more into one PR than necessary.